src/menu.scss

```diff
@@ -5,165 +5,295 @@
+      .#{$block}__title {
+        color: $fd-menu-item-color;
+      }
+
+      background-color: $fd-menu-item-background-color--selected;
+      border-bottom: var(--sapList_BorderWidth) solid var(--sapList_SelectionBorderColor);
+
+      .#{$block}__addon-before,
+      .#{$block}__addon-after {
+        color: $fd-menu-primary-icon-color;
+      }
 
       @include fd-hover() {
-        @include fd-var-color("background-color", fd-color-state("selected-hover"), --fd-color-background-selected-hover);
+        background-color: $fd-menu-item-background-color--selected-hover;
       }
     }
 
-    @include fd-active-pressed-selected() {
-      @include fd-var-color("color", fd-color-state("selected", "action"), --fd-color-action-selected);
+    @include fd-fiori-focus();
 
-      outline: none;
+    @include fd-disabled() {
+      opacity: var(--sapContent_DisabledOpacity);
+      cursor: not-allowed;
+      text-decoration: none;
+      pointer-events: none;
     }
 
-    &:focus {
-      outline-style: dotted;
-      outline-width: 1px;
+    .#{$block}__title {
+      @include fd-reset();
 
-      @include fd-var-color("outline-color", fd-color-state("hover", "action"), --fd-color-action-focus);
-    }
+      flex: 3 3 10%;  // needed to keep the icons aligned
+      color: $fd-menu-item-color;
 
-    @include fd-disabled() {
-      @include fd-var-color("outline-color", fd-color-state("disabled", "action"), --fd-color-action-disabled);
-      @include fd-var-color("color", $fd-forms-color--disabled, --fd-color-text-4);
+      @include fd-weight("normal");
 
-      cursor: not-allowed;
+      font-size: var(--sapFontSize);
+      font-family: var(--sapFontFamily);
       text-decoration: none;
+
+      @include fd-ellipsis();
     }
   }
 
-  &--addon-before {
-    .fd-menu__item {
-      padding-top: $fd-menu-item-padding-y;
-      padding-bottom: $fd-menu-item-padding-y;
-      padding-left: 2px;
-      padding-right: 2px;
+  &__addon-before {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    // cozy tablet icon size
+    width: 2.25rem; // to account for item right padding of 16px or 1rem so 2.25rem-1rem = 1.25rem
+    height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
+    margin: 0 0 0 -0.75rem;
+
+    @include fd-rtl() {
+      margin: 0 -1rem 0 0; // since before and after icons switch places, their margins also switch
     }
+
+    color: $fd-menu-primary-icon-color;
+
+    @include fd-active() {
+      color: $fd-menu-item-color--active;
+    }
+
+    @include fd-selected() {
+      color: $fd-menu-primary-icon-color;
+
+      @include fd-hover() {
+        color: $fd-menu-item-color--active;
+      }
+    }
+
+    display: flex;
+    background-color: transparent;
+
+    @include fd-icon-size("m", "before");
+    @include fd-icon-size("m", "after");
```

Comment:
This mixin makes the icon 16px. We should use `rem`
<img width="496" alt="Screen Shot 2020-03-09 at 1 50 57 PM" src="https://user-images.githubusercontent.com/39598672/76242433-200c5f00-620d-11ea-98e5-33cb418f9907.png">
<img width="264" alt="Screen Shot 2020-03-09 at 1 50 49 PM" src="https://user-images.githubusercontent.com/39598672/76242439-200c5f00-620d-11ea-84c4-03c5e02ca021.png">

Reply:
Okay, I will just use `var(--sapFontLargeSize)` directly instead